pkg/gopass/secret/kv.go

```diff
@@ -21,7 +21,55 @@ func NewKV() *KV {
 	}
 }
 
-// KV is a simple key value secret
+// NewKVWithData returns a new KV secret populated with data
+func NewKVWithData(pw string, kvps map[string]string, body string, converted bool) *KV {
+	kv := &KV{
+		password: pw,
+		data:     kvps,
```

Comment:
`make(map[string]string)` would be better for readability, no? Since you need to do an explicit copy of each value, which is done below.

Reply:
Yes, indeed. This was left over from a previous iteration.